production-marathon/README.md

```diff
@@ -1,20 +1,22 @@
 # Production Marathon Deployment
 
-The Production Marathon deployment supports the deployment of the CAF Job Service on Mesos/Marathon. This folder contains the marathon environment and template files that are required to deploy the service application.
+The Production Marathon deployment supports the deployment of the Job Service on Mesos/Marathon. This folder contains the marathon environment and template files that are required to deploy both the Job Service and Job Tracking applications.
```

Comment:
Job Tracking Worker. Same goes for any further references to Job Tracking in the README.